src/food/custom-foods.ts

```diff
@@ -0,0 +1,40 @@
+import { Material } from 'org.bukkit';
+import { PlayerItemConsumeEvent } from 'org.bukkit.event.player';
+import { EquipmentSlot, ItemStack } from 'org.bukkit.inventory';
+import { giveItem } from '../common/helpers/inventory';
+import { CustomFoodInfo } from './FoodInfo';
+import { addFoodPoints, addSaturation } from './helpers';
+
+const CUSTOM_FOOD_MATERIAL = Material.POISONOUS_POTATO;
+
+export function isCustomFood(item: ItemStack) {
+  if (item.type !== CUSTOM_FOOD_MATERIAL) return false;
+  if (!item.itemMeta.hasCustomModelData()) return false;
```

Comment:
```suggestion
  return item.itemMeta.hasCustomModelData();
```
and remove the following return.